metrics/resource_view.go

```diff
@@ -59,6 +63,38 @@ type meters struct {
 var resourceViews = storedViews{}
 var allMeters = meters{
 	meters: map[string]*meterExporter{"": &defaultMeter},
+	clock:  cleanupClock,
+}
+
+var cleanupClock = clock.Clock(clock.RealClock{}) // Var needed to avoid allMeters->defaultMeter->allMeters.clock cycle
+var cleanupOnce sync.Once
+var meterExporterScanFrequency = 1 * time.Minute
+var maxMeterExporterAge = 10 * time.Minute
```

Comment:
```suggestion
var (
resourceViews = storedViews{}
allMeters = meters{
	meters: map[string]*meterExporter{"": &defaultMeter},
	clock:  cleanupClock,
}

cleanupClock = clock.Clock(clock.RealClock{}) // Var needed to avoid allMeters->defaultMeter->allMeters.clock cycle
cleanupOnce sync.Once
meterExporterScanFrequency = 1 * time.Minute
maxMeterExporterAge = 10 * time.Minute
)
```

Reply:
Done.